executor/common.h

```diff
@@ -205,7 +205,7 @@ static void use_temporary_dir(void)
 #include <sys/stat.h>
 #include <sys/types.h>
 
-static void remove_dir(const char* dir)
+static void __attribute__((noinline)) remove_dir(const char* dir)
```

Comment:
This is gcc-ism and will break msvc.
We don't have any precedents of using attributes in common*.h code yet. So I am not sure what's the right way to do it. If we use any macros, it also needs to play well with pkg/csource.
Also needs a comment. __attribute__((noinline)) is never a normal, self-explainable thing.

Reply:
thanks for feedback, as i understood it, malloc for filename buffer is out of the question, right ?

Reply:
Yes. But these names are also short. So we are trying to remove the warning, not fix a bug.

Reply:
isn't remove_dir i changed for POSIX only, does msvc support *opendir* and co ?

Reply:
done